packages/opencode/script/ppt-quality-eval/route-skills/svg-pptx-native/scripts/svg_to_pptx/pptx_package/discovery.py

```
@@ -0,0 +1,101 @@
+            match = re.search(r'slide[_]?(\d+)', stem)
+            if match:
+                index = int(match.group(1))
+                mapped_stem = svg_index_mapping.get(index)
+                if mapped_stem:
+                    notes[mapped_stem] = content
+
+            # Filename-based matching (overrides index-based)
+            if stem in svg_stems_mapping:
+                notes[stem] = content
+        except Exception:
+            pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)